shared/media-accelerator/develop/multi-channel-streaming/project-implementation/android.mdx

```diff
@@ -0,0 +1,340 @@
+        showMessage("Failed to create connection 1");
+    } else {
+        showMessage("Connection1 created");
+    }
+
+    // Create a second connection
+    connectionId2 = agoraRtcService.createConnection();
+    if (connectionId2 == AgoraRtcService.ConnectionIdSpecial.CONNECTION_ID_INVALID) {
+        showMessage("Failed to create connection 2");
+    } else {
+        showMessage("Connection2 created");
+    }
+    ```
+
+1. **Join multiple channels**
+
+    When a user presses the **Join** button, you join two channels. To join a second channel, add the following code at the end of `joinChannel(View view)`:
+
+    ```java
+    // Join a second channel
+    ret = agoraRtcService.joinChannel(connectionId2, channelName2,
+            uid, token2, channelOptions);
+    if (ret != AgoraRtcService.ErrorCode.ERR_OKAY) {
+        showMessage("Join channel2 failure!");
+        isJoined2 = false;
+    } else {
+        isJoined2 = true;
+    }
+    ```
+
+1. **Set joined status**
+
+    When your call to `agoraRtcService.joinChannel` succeeds, you receive a notification through the `onJoinChannelSuccess` callback. You use connection Id to identify the channel and set the corresponding `isJoined` variable to true. To do this, **replace** the `onJoinChannelSuccess` method under `agoraRtcEvents` with the following: 
+
+    ```java
+    @Override
+    public void onJoinChannelSuccess(int connId, int uid, int elapsed_ms) {
+        if (connId == connectionId) {
+            // Successfully joined channel 1
+            isJoined = true;
+            showMessage("Successfully joined channel " + channelName);
+        } else if (connId == connectionId2) {
+            // Successfully joined channel 2
+            isJoined2 = true;
+            showMessage("Successfully joined channel " + channelName2);
+        }
+    }
+    ```
+
+1. **Stream audio to the first channel**
+
+    In this example, you stream audio to the first channel and video to the second channel. To start or stop the audio stream when a user taps the **Start stream 1** button, add the following method to the `MainActivity` class:
+
+    ```java
+    public void stream1(View view) {
+        Button button = (Button) view;
+        
+        if (!isJoined) {
+            showMessage("Join a channel first");
+            return;
+        } else if (audioThread != null && audioThread.isAlive()) {
+            audioThread.sendStop();
+            audioThread = null;
+            button.setText("Start stream 1");
+            return;
+        }
+        // Create a thread to send audio frames
+        audioThread = new AudioSendThread(getApplicationContext(), agoraRtcService,
+                channelName, connectionId, 16000, 1, 2);
+        audioThread.sendStart();
+        showMessage("Audio thread started on channel " + channelName);
+        button.setText("Stop audio stream on channel " + channelName);
+    }
+    ```
+
+1. **Stream video to the second channel**
```

Comment:
stream to the video channel

Reply:
Since this is a multi-channel doc, the important thing is that we have a second channel. 
You can send audio, video, or both over the second channel so we can't really call it the video channel.